CMakeLists.txt

```diff
@@ -202,6 +202,12 @@ git_submodule_init(
   CHECK_FILE "3rdparty/edbee-lib/CMakeLists.txt" SUBMODULE_PATH
   "3rdparty/edbee-lib" READABLE_NAME "edbee-lib editor widget")
 
+if(APPLE OR WIN32)
```

Comment:
You need to import the sort of changes that were made for the MacOS one here (although we don't use CMake for Windows CI/CB builds currently. It also need to be made conditional on a USE_SENTRY like CMake variable - which is extended to both MacOs and Windows cases now I see.